packages/field-base/src/styles/field-base-styles.js

```diff
@@ -100,7 +174,10 @@ export const field = css`
     font-weight: var(--vaadin-input-field-error-font-weight, 400);
     color: var(--vaadin-input-field-error-color, var(--vaadin-text-color));
     display: flex;
-    gap: var(--vaadin-gap-s);
+    gap: var(--vaadin-gap-xs);
```

Comment:
Extracted this into https://github.com/vaadin/web-components/pull/10462 along with relevant screenshot updates, to make this PR focused on alignment.